packages/core/app/src/initialize.ts

```diff
@@ -1,53 +1,42 @@
-import type { NavigatorWalletsWindow, Wallet, WindowNavigatorWalletsPushCallback } from '@wallet-standard/base';
+import type {
+    AppInitializeEvent as AppInitializeEventInterface,
+    AppInitializeEventAPI,
+    Wallet,
+    WalletEventsWindow,
+} from '@wallet-standard/base';
 
-declare const window: NavigatorWalletsWindow;
-
-let initializedWallets: InitializedWallets | undefined = undefined;
+declare const window: WalletEventsWindow | undefined;
 
 /**
  * TODO: docs
  */
-export function initializeWindowNavigatorWallets(): InitializedWallets {
-    // If we've already initialized, just return.
+export function initializeApp(): InitializedWallets {
```

Comment:
Is this more like `getWalletStandardAPI` now? Or something else? There's just something weird about:

```typescript
const wallets = initializeApp();  // One might presume that `wallets` is an array now.
```

Also, the caller dgaf if they're initializing it or if they're just getting the already-initialized API, and they shouldn't know, and they shouldn't care.

Reply:
I kind of just kept this convention for parity across both wallet and app -- they are two.. fists of the same fist bump.

I will say, `getWalletStandardAPI` where it has the side effect of setting up event listeners and dispatching events, feels semantically wrong to me.

`initialize` is kind of a placeholder word, but I want to express that it's doing an imperative thing (could use another word).

Reply:
I don’t think I mind the side effect. It’s a lazy initializing memoized getter.